src/main/resources/application-prod-gcp.properties

```diff
@@ -10,3 +10,6 @@ hentinntektlistebolk.scopes=api://prod-fss.team-inntekt.inntektskomponenten/.def
 organisasjon.rs.url=https://ereg-services.prod-fss-pub.nais.io/api/v2/organisasjon
 
 altinn.scopes=api://prod-gcp.arbeidsgiver.altinn-rettigheter-proxy/.default
+
+dokarkiv.base.url=http://dokarkiv.teamdokumenthandtering/rest/journalpostapi/v1/journalpost
```

Comment:
base.url kunne egentlig flyttes opp til application.properties siden den er lik for dev og prod.

Reply:
https://dokarkiv.prod-fss-pub.nais.io. husk legge inn i nais policy

Reply:
Lagt inn!